templates/emails/plain/email-order-details.php

```diff
@@ -12,37 +12,38 @@
  *
  * @see https://docs.woocommerce.com/document/template-structure/
  * @package WooCommerce/Templates/Emails
- * @version 3.5.0
+ * @version 3.7.0
  */
 
-if ( ! defined( 'ABSPATH' ) ) {
-	exit;
-}
+defined( 'ABSPATH' ) || exit;
 
 do_action( 'woocommerce_email_before_order_table', $order, $sent_to_admin, $plain_text, $email );
 
 /* translators: %1$s: Order ID. %2$s: Order date */
 echo wp_kses_post( wc_strtoupper( sprintf( __( '[Order #%1$s] (%2$s)', 'woocommerce' ), $order->get_order_number(), wc_format_datetime( $order->get_date_created() ) ) ) ) . "\n";
-echo "\n" . wc_get_email_order_items( $order, array( // WPCS: XSS ok.
-	'show_sku'      => $sent_to_admin,
-	'show_image'    => false,
-	'image_size'    => array( 32, 32 ),
-	'plain_text'    => true,
-	'sent_to_admin' => $sent_to_admin,
-) );
+echo "\n" . wc_get_email_order_items( // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+	$order,
+	array(
+		'show_sku'      => $sent_to_admin,
+		'show_image'    => false,
+		'image_size'    => array( 32, 32 ),
+		'plain_text'    => true,
+		'sent_to_admin' => $sent_to_admin,
+	)
+);
 
 echo "==========\n\n";
 
-$totals = $order->get_order_item_totals();
+$item_totals = $order->get_order_item_totals();
 
-if ( $totals ) {
-	foreach ( $totals as $total ) {
+if ( $item_totals ) {
+	foreach ( $item_totals as $total ) {
 		echo wp_kses_post( $total['label'] . "\t " . $total['value'] ) . "\n";
 	}
 }
 
 if ( $order->get_customer_note() ) {
-	echo esc_html__( 'Note:', 'woocommerce' ) . "\t " . wp_kses_post( wptexturize( $order->get_customer_note() ) ) . "\n";
+	echo esc_html__( 'Note:', 'woocommerce' ) . "\t " . wp_kses_post( nl2br( wptexturize( $order->get_customer_note() ) ) ) . "\n";
```

Comment:
This change will cause the text <br/> to be outputted in text emails. I think it should not be passed through nl2br for plain text emails.

Reply:
Ohh, yeah, we can avoid that, still plain text emails should strip those anyway.